ironfish-rust/src/errors.rs

```diff
@@ -2,19 +2,26 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
 
+use std::backtrace::Backtrace;
 use std::error::Error;
 use std::fmt;
 use std::io;
 use std::num;
 use std::string;
 
+#[derive(Debug)]
+pub struct IronfishError {
+    pub kind: IronfishErrorKind,
+    pub backtrace: Backtrace,
```

Comment:
This field is not read anywhere (unless I'm mistaken), so how is one supposed to see this backtrace?

Reply:
It is used here:

https://github.com/iron-fish/ironfish/blob/b887f5198c6b7c9b9d06eddbc7f72a99d5650c98/ironfish-rust-nodejs/src/lib.rs#L22-L24